src/metpy/calc/exceptions.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains specific exceptions raised by thermodynamic calculations."""
```

Comment:
While it's only currently used by something in thermo, could be anywhere in calc:
```suggestion
"""Contains specific exceptions raised by calculations."""
```

Reply:
Done